lib/liquid/tags/include.rb

```diff
@@ -101,6 +84,61 @@ def render_to_output_buffer(context, output)
     alias_method :parse_context, :options
     private :parse_context
 
+    def rigid_parse(markup)
+      p = @parse_context.new_parser(markup)
+
+      @template_name_expr = safe_parse_expression(p)
+      with_or_for = p.id?("for") || p.id?("with") || nil
+      @variable_name_expr = nil
+      if with_or_for
+        @variable_name_expr = safe_parse_expression(p)
+      end
+
+      alias_name = nil
+      if p.id?("as")
+        alias_name = p.consume(:id)
+      end
+
+      @alias_name = alias_name
+
+      # optional comma
+      p.consume?(:comma)
+
+      @attributes = {}
+      while p.look(:id)
+        key = p.consume
+        p.consume(:colon)
+        @attributes[key] = safe_parse_expression(p)
+        p.consume?(:comma) # optional comma
+      end
+
+      p.consume(:end_of_string)
+    end
+
+    def strict_parse(markup)
+      lax_parse(markup)
+    end
+
+    def lax_parse(markup)
+      if markup =~ SYNTAX
+
```

Comment:
[nit] Shouldn't rubocop have caught this empty line 🤔 
```suggestion
```

Reply:
Personally, I think rubocop shouldn't remove this line. But, I've removed it ;)